services/src/main/java/org/broadleafcommerce/payment/service/gateway/DefaultPayPalCheckoutTransactionService.java

```diff
@@ -140,21 +155,22 @@ public PaymentResponse authorize(PaymentRequest paymentRequest) {
 
     @Override
     public PaymentResponse capture(PaymentRequest paymentRequest) {
-        PaymentResponse paymentResponse =
-                new PaymentResponse(DefaultPaymentTypes.THIRD_PARTY_ACCOUNT,
-                        PayPalCheckoutPaymentGatewayType.PAYPAL_CHECKOUT)
-                                .transactionType(DefaultTransactionTypes.CAPTURE);
+        PaymentResponse paymentResponse = paymentResponseUtil.buildPaymentResponse(paymentRequest,
+                PayPalCheckoutPaymentGatewayType.PAYPAL_CHECKOUT, DefaultTransactionTypes.CAPTURE);
```

Comment:
We should move the `paymentResponseUtil.buildPaymentResponse(...)` call inside of the `try` block & we should catch `Exception` instead of `PaymentException`, to guarantee that this method will always return a PaymentResponse